FemtoVGRenderer is no longer available

### Bug Description

Enabling `slint/renderer-femtovg` does not work in 1.16.

```
error[E0432]: unresolved import `i_slint_renderer_femtovg::FemtoVGOpenGLRenderer`
   --> slint-1.16.1/lib.rs:417:17
    |
417 |         pub use i_slint_renderer_femtovg::FemtoVGOpenGLRenderer as FemtoVGRenderer;
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `FemtoVGOpenGLRenderer` in the root
    |
note: found an item that was configured out
   --> i-slint-renderer-femtovg-1.16.1/lib.rs:584:10
    |
583 | #[cfg(feature = "opengl")]
    |       ------------------ the item is gated behind the `opengl` feature
584 | pub type FemtoVGOpenGLRenderer = FemtoVGRenderer<opengl::OpenGLBackend>;
    |          ^^^^^^^^^^^^^^^^^^^^^
help: a similar name exists in the module
    |
417 -         pub use i_slint_renderer_femtovg::FemtoVGOpenGLRenderer as FemtoVGRenderer;
417 +         pub use i_slint_renderer_femtovg::FemtoVGRenderer as FemtoVGRenderer;
    |

error[E0432]: unresolved import `i_slint_renderer_femtovg::opengl`
   --> slint-1.16.1/lib.rs:422:43
    |
422 |         pub use i_slint_renderer_femtovg::opengl::OpenGLInterface;
    |                                           ^^^^^^ could not find `opengl` in `i_slint_renderer_femtovg`
    |
note: found an item that was configured out
   --> i-slint-renderer-femtovg-1.16.1/lib.rs:41:9
    |
 40 | #[cfg(feature = "opengl")]
    |       ------------------ the item is gated behind the `opengl` feature
 41 | pub mod opengl;
    |         ^^^^^^
```

### Reproducible Code (if applicable)

```slint

```

### Environment Details

- Slint Version: 1.16.1
- Platform/OS:
- Programming Language: rust
- Backend/Renderer:


### Product Impact

_No response_

--- a/api/rs/slint/Cargo.toml
+++ b/api/rs/slint/Cargo.toml
@@ -157,7 +157,7 @@ renderer-winit-skia-vulkan = ["renderer-skia-vulkan"]
 renderer-winit-software = ["renderer-software"]
 
 ## Render using the [FemtoVG](https://crates.io/crates/femtovg) crate.
-renderer-femtovg = ["i-slint-backend-selector/renderer-femtovg", "dep:i-slint-renderer-femtovg", "std"]
+renderer-femtovg = ["i-slint-backend-selector/renderer-femtovg", "i-slint-renderer-femtovg/opengl", "std"]
 ## Render using the [FemtoVG](https://crates.io/crates/femtovg) crate and [WGPU](https://crates.io/crates/wgpu).
 renderer-femtovg-wgpu = ["i-slint-backend-selector/renderer-femtovg-wgpu", "i-slint-renderer-femtovg/wgpu", "std"]
 

